pkg/utils/controller/metrics/metrics.go

```diff
@@ -61,6 +61,13 @@ var (
 		Name: "fleet_workload_eviction_complete",
 		Help: "Eviction complete status ",
 	}, []string{"name", "isCompleted", "isValid"})
+
+	// FleetUpdateRunStatusLastTimestampSeconds is a prometheus metric which holds the
+	// last update timestamp of update run status in seconds.
+	FleetUpdateRunStatusLastTimestampSeconds = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "fleet_workload_update_run_status_last_timestamp_seconds",
+		Help: "Last update timestamp of update run status in seconds",
+	}, []string{"name", "status"})
```

Comment:
include the generation? we may support updating updateRun in the future.

Reply:
I'm not sure if we really need the generation in metrics. If there's something wrong, we usually leverage logs to debug not the metrics. When setting up alerts, I cannot think of a scenario where I need to filter or do something specific for the generation. And generation itself adds more cardinality.

Reply:
add generation in metrics.